libs/books/data-access/src/lib/books-data-access.module.constants.ts

```diff
@@ -0,0 +1,22 @@
+export const BOOKS_DATA_ACCESS_CONSTANTS = {
+    READING_LIST_API: '/api/reading-list',
+    SNACK_BAR: {
+      ADD_MESSAGE: 'added to reading list',
+      DELETE_MESSAGE: 'removed from reading list',
+      ACTION_TEXT: 'Undo',
+      HORIZONTAL_POSITION: 'end',
+      VERTICAL_POSITION: 'bottom',
+      DURATION: 7000
```

Comment:
I believe that 7 seconds is too much time. You are correct regarding the Material Desing standards (https://material.io/components/snackbars#behavior) _"a minimum of four seconds, and a maximum of ten seconds"_, but I believe that 5 seconds is enough in this case, since the text in the snackbar is not that long, in any case.

Reply:
Hi Katerina,

Thanks for the input and your point is noted for future reference.